portfolio/src/main/java/com/google/sps/servlets/models/Comment.java

```diff
@@ -7,31 +7,43 @@
 public class Comment {
   private long id;
   private long timestamp;
+  private String userId;
   private String username;
   private String content;
+  private int likes;
 
-  public Comment(long id, long timestamp, String username, String content) {
+  public Comment(
+    long id, long timestamp, 
+    String userId, String username, String content, 
+    int likes
+  ) {
     this.id = id;
     this.timestamp = timestamp;
+    this.userId = userId;
     this.username = username;
     this.content = content;
+    this.likes = likes;
   }
 
   public static Comment convertEntityToComment(Entity entity) {
     long id = entity.getKey().getId();
     long timestamp = (long) entity.getProperty("timestamp");
+    String userId = (String) entity.getProperty("userId");
     String username = (String) entity.getProperty("username");
     String content = (String) entity.getProperty("content");
+    int likes = (int) (long) entity.getProperty("likes");
```

Comment:
(long) should be unnecessary

Reply:
for some reason the DB stores my 'likes' field as Long, so if I don't have that (long) there it's gonna throw errors

Reply:
According to the Datastore documentation, this is normal behavior since type `int` would automatically be converted to database's native type -> long.

![Screenshot 2020-06-05 at 2 56 39 PM](https://user-images.githubusercontent.com/25425710/83913024-dca68b80-a73c-11ea-8650-9538f7d36e02.png)